integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/script/InlineNashornJsFunctionsForBothInAndOutTestCase.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.esb.mediator.test.script;
+
+import org.apache.axiom.om.OMElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+import javax.xml.namespace.QName;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+/**
+ * This test case verifies that, mediation with inline NashornJs scripts happens correctly.
+ */
+public class InlineNashornJsFunctionsForBothInAndOutTestCase extends ESBIntegrationTest {
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        super.init();
+    }
+
+    @Test(groups = "wso2.esb", description = "Invoke a sequence which has inlined Nashornjs functions for both 'in' "
+            + "and outMediator' flows")
+    public void testInvokeSequence() throws Exception {
+        OMElement response = axis2Client.sendCustomQuoteRequest(getProxyServiceURLHttp
+                ("nashornJsInAndOutTestProxy"), null, "inlineTest");
+        assertNotNull(response, "Fault response message null");
+        assertNotNull(response.getQName().getLocalPart(), "Fault response null localpart");
+        assertEquals(response.getQName().getLocalPart(), "CheckPriceResponse", "Fault localpart mismatched");
+        assertNotNull(response.getFirstElement().getQName().getLocalPart(), " Fault response null localpart");
+        assertEquals(response.getFirstElement().getQName().getLocalPart(), "Code", "Fault localpart mismatched");
+        assertEquals(response.getFirstElement().getText(), "inlineTest", "Fault value mismatched");
+        assertNotNull(response.getFirstChildWithName(new QName("http://services.samples/xsd", "Price")), "Fault "
+                + "response null localpart");
+    }
```

Comment:
Shall we put "Fault response null localpart" into a single line, and then break the statement into two at the comma.